packages/cosmic-proto/package.json

```diff
@@ -96,6 +100,10 @@
       "types": "./dist/codegen/icq/v1/packet.d.ts",
       "default": "./dist/codegen/icq/v1/packet.js"
     },
+    "./stride/stakeibc/tx.js": {
```

Comment:
@turadg I think, we should also add `stride/staketia/tx` and `stride/stakedym/tx` as some of it will be used in the future.